SpywareFilter/sections/tracking_servers.txt

```diff
@@ -54,6 +54,7 @@
 !
 ||reports.koalametrics.com^
 ||js.go2sdk.com^
+||srvr.dmvs-apac.com^
```

Comment:
It also blocks embedded Dailymotion videos on some websites.

<details>
 <summary>Screenshot</summary>

![resim](https://user-images.githubusercontent.com/104112959/170650406-3d34b52e-b6cb-4878-bb1f-0b4a937e405c.png)


</details>

```
https://www.preview.ph/
https://www.esquiremag.ph/culture/movies-and-tv/stranger-things-season-4-volume-1-review-a2384-20220527-src-esquire?ref=home_feed_1
https://www.smartparenting.com.ph/
https://www.femalenetwork.com/
```

Reply:
And other than that, can you also add the following rules for right/left ad spaces?

```
cumhuriyet.com.tr##.page-left-fixed-ad
cumhuriyet.com.tr##.page-right-fixed-ad
```

Reply:
I don't think that we should block it.
However, this works fine on my end.
```
||static1.dmcdn.net/playerv5/dmp.pes_pip_tracking.*.js
```

[For this](https://github.com/AdguardTeam/AdguardFilters/pull/119956/files#r883357299), one rule can be enough
```
cumhuriyet.com.tr##div[class^="page-"][class*="fixed-ad"]
```